### DISCUSSION TITLE
Training time

### DISCUSSION THREAD

**Discussion by ConstantinaNicolaou** [SalesforceAIResearch/uni2ts#85] [Q&A]
Hi, I was wondering how long it took to pre-train each of the three MOIRAI models on the LOTSA dataset. I could only find the infrastructure used (NVIDIA A100-40G GPUs) in the paper but not the training time (apologies if you have stated this somewhere and I have missed it). I would appreciate it if you could please confirm the infrastructure used and provide an estimate on the training time. Thank you!

**Comment by ConstantinaNicolaou** [comment]
Hey, just a quick reminder on the above message in case it fell through the cracks. I would really appreciate a rough estimate of training time and infrastructure used. Thank you.

**Comment by gorold** [comment]
Hi, don't think I can give a good estimate as the training of the current models were not optimized previously. You could run your own controlled experiments to get a good measure of the train time required.
